pkgs/by-name/gi/gittuf/package.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, fetchFromGitHub, buildGoModule }:
+
+buildGoModule rec {
+  pname = "gittuf";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "gittuf";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-lECvgagcqBS+BVD296e6WjjSCA3vI0nfLzpLTi/7N0I=";
+  };
+
+  vendorHash = "sha256-UKhXbZXKNtMnQe7sHBOmzzXGBHuDTYeZGKnteZirskA=";
+
+  doCheck = false;
+
+  meta = with lib; {
+    description = "A security layer for Git repositories";
+    homepage = "https://gittuf.dev";
+    license = licenses.asl20;
+    platforms = platforms.all;
```

Comment:
```suggestion
```

Reply:
https://ryantm.github.io/nixpkgs/stdenv/meta/ says 

> If no platform is specified, the package does not have prebuilt binaries.

could you explain why you'd rather remove the attribute?

Reply:
It's set by default for Go modules, see https://github.com/NixOS/nixpkgs/blob/4cf45689d470c5dbf83f9b00f25dd670237efe0d/pkgs/build-support/go/module.nix#L302

Reply:
interesting, thanks!